### DISCUSSION TITLE
Implementation of karma protocol, how does it work

### DISCUSSION THREAD

**Discussion by IMFefer** [celo-org/social-connect#213] [Help (Q&A)] [answered]
My team and I were looking to implement the karma protocol but cant quite understand how to start using it in our dapp

**Comment by arthurgousset** [comment, marked as answer]
Hey @IMFefer 👋 Thanks for posting here. Could you link me to the "karma protocol" please. I'm not sure I'm following what you mean. Thanks! 🙏

**Reply by IMFefer** [reply]
https://www.karma-protocol.co/

**Reply by arthurgousset** [reply]
Sorry this is a separate project and not part of the celo-monorepo. Perhaps you can contact them directly via the website [info@karma-protocol.co](mailto:info@karma-protocol.co)?
